.github/workflows/ci.yml

```diff
@@ -40,6 +40,29 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --workspace --tests --examples
 
+  docs:
+    name: Rust doc comments
+    runs-on: ubunut-latest
+    steps:
```

Comment:
This causes error
```
The workflow is not valid. .github/workflows/ci.yml (Line: 46, Col: 11): Unexpected value '' .github/workflows/ci.yml (Line: 49, Col: 5): 'steps' is already defined
```
You can see it on the "Actions" tab: https://github.com/deltachat/deltachat-core-rust/actions/runs/503059698

Reply:
doh, yes.  Thanks!